reports/Alabama.md

```diff
@@ -4,9 +4,11 @@
 
 Police arrested two AL.com journalists covering a protest at 7:35 on June 3. There was a 7PM curfew in effect, which exempted journalists who were carrying a letter from their media outlet; both arrested journalists were carrying such a letter and had press lanyards. Both journalists were released shortly thereafter; while in custody, they were photographed. Birmingham police later said that the arrest was made for the journalists' safety.
 
+Two employees of Alabama Media Group were restrained using zip ties.
```

Comment:
Is this in the original video?

Reply:
There is no video of the arrest, the only video linked is a short clip of protestors attacking the reporters and vandalizing a storefront.

The article states:
> The two were restrained using zip ties. Koplowitz and Dunigan were placed in a transport van and taken to the city jail along with several other protestors who had been arrested. According to Koplowitz, the entire group was taken out of the van and handcuffed to a bench when they arrived at the city jail.